### PR TITLE
[JA] Translate nerves 1.0.0

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/advanced/nerves` section.
Please review! @elixirschool/japanese

### CODE REVIEW

File: ja/lessons/advanced/nerves.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  end
+
+  def disable() do
+    GenServer.cast(__MODULE__, :disable)
+  end
+end
+
+```
+
+これを有効にするには、それを `lib/network_led/application.ex` の監視ツリーに追加する必要があります。`def children(_target) do` グループの下に `{NetworkLed.Blinker, name: NetworkLed.Blinker}` を追加してください。
```
Comment (starfish719) [inline ja/lessons/advanced/nerves.md]: `supervision tree` is Japanese `監視ツリー` 🤔 
I feel of little wrongness.

Comment (riseshia) [reply ja/lessons/advanced/nerves.md]: I think スーパーバイザーツリー is enough, do you want to translate this term?

File: ja/lessons/advanced/nerves.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+これで、メモリカードをデバイスに挿入して動作するかどうかを確認します。
+
+画面が接続されている場合、このメモリカードを挿入した状態でデバイスの電源を入れた後に、Linuxの起動シーケンスが表示されるはずです。
+
+## ネットワークの設定
+
+次のステップはネットワークを設定することです。Nervesのエコシステムはさまざまなパッケージを提供しています。[nerves_network](https://github.com/nerves-project/nerves_network) は、有線イーサネットポートを介してデバイスをネットワークに接続するために必要なものです。
+
+あなたのプロジェクトにはすでに `nerves_init_gadget` が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ `config：nerves_init_gadget`を実行した後は` config / config.exs`の中の設定を見てください）。静的IPアドレスを持つ方が簡単です。
```
Comment (riseshia) [inline ja/lessons/advanced/nerves.md]:
```suggestion
あなたのプロジェクトにはすでに `nerves_init_gadget` が依存関係として存在しています。しかし、デフォルトではDHCPを使います（`config/config.exs`の中の `config :nerves_init_gadget`の次に移動してください）。静的IPアドレスを持つ方が簡単です。
```
I'm not sure, but one thing I can say is "`config：nerves_init_gadget`を実行した後は` config / config.exs`の中の設定を見てください`" is wrong definitely 🤔

File: ja/lessons/advanced/nerves.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
+---
+version: 1.0.0
+title: Nerves
+---
+
+## はじめに
+
+このレッスンではNervesについて話します。 Nervesプロジェクトは組込みソフトウェア開発にElixirを使用するためのフレームワークです。Nervesのウェブサイトにあるように、それはあなたが「Elixirで、安全な組み込みソフトウェアを作り、そして展開する」ことを可能にします。このレッスンは他のElixir Schoolのレッスンとは少し異なります。上級者向けのシステム設定と追加のハードウェアの両方が必要なため、Nervesを習得するのは少し困難です。したがって、初心者にはオススメしません。
+
+Nervesを使ってコードを書くためには、ネットワークによってこのデバイスにアクセスするための有線ネットワーク接続として、 [supported targets](https://hexdocs.pm/nerves/targets.html) のいずれか、お好みのハードウェアでサポートされているメモリカードを持ったカードリーダーが必要です。
```
[the remark this answers is not in the document]
Comment (starfish719) [reply ja/lessons/advanced/nerves.md]: @riseshia https://github.com/elixirschool/elixirschool/pull/1889/commits/9064e618be6f31645253f18de32a66b92dcfd57d 👍

File: ja/lessons/advanced/nerves.md (hunk 2 above)
Comment (starfish719) [reply ja/lessons/advanced/nerves.md]: @riseshia https://github.com/elixirschool/elixirschool/pull/1889/commits/6f620e5719c16fd2e329c7e3ea05521d35f93b82

File: ja/lessons/advanced/nerves.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+
+## ネットワークファームウェアの書き込み
+
+これまでのところ、私達はSDカードに書き込んで、物理的にそれらを私達のハードウェアにロードしてきました。これは最初は問題ありませんが、ネットワーク経由で更新する方が簡単です。`nerves_firmware_ssh` パッケージはまさにそれをしてくれます。デフォルトでは、すでにプロジェクトに存在し、ディレクトリ内のSSHキーを自動検出して見つけるように設定されています。
+
+ネットワークファームウェアアップデート機能を使うためには、 `mix firmware.gen.script` を通してアップロードスクリプトを生成する必要があります。このコマンドはファームウェアを更新するために実行できる新しい `upload.sh` スクリプトを生成します。
+
+前の手順を実行してネットワークが機能している場合は、問題ないので次に進みます。
+
+設定を更新するための最も簡単な方法は `mix firmware && ./upload.sh 192.168.88.2` を使うことです。最初のコマンドは更新されたファームウェアを作成し、二番目のコマンドはそれをネットワークにプッシュしてデバイスを再起動します。 SDカードをデバイスに出し入れする必要がなくなります。
```
Comment (riseshia) [inline ja/lessons/advanced/nerves.md]:
```suggestion
設定を更新するための最も簡単な方法は `mix firmware && ./upload.sh 192.168.88.2` を使うことです。最初のコマンドは更新されたファームウェアを作成し、二番目のコマンドはそれをネットワーク経由でプッシュしてデバイスを再起動します。 SDカードをデバイスに出し入れする必要がなくなります。
```

File: ja/lessons/advanced/nerves.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+
+_トラブルシューティング：ホームフォルダにsshキーが存在しない場合は、 `No SSH public keys found in ~/.ssh.` というエラーが表示されます。この場合、 `ssh-keygen` を実行してネットワークアップデート機能を使用するためにファームウェアを再書き込みする必要があります。_
+
+## LEDのコントロールを設定する
+
+LEDとやり取りするには、[nerves_leds](https://github.com/nerves-project/nerves_leds) パッケージをインストールする必要があります。これは、`mix.exs` ファイルに `{:nerves_leds, "~> 0.8", targets: @all_targets},` を追加することによって行われます。 
+
+依存関係を設定したら、特定のデバイスのLEDリストを設定する必要があります。例えば、すべてのRaspberry Piモデルのために、ただ一つのLEDである `led0` が搭載されています。 `config/config.exs` に `config :nerves_leds, names: [green: "led0"]` を追加して使用しましょう。
+
+他のデバイスについては、[corresponding part of the nerves_examples project](https://github.com/nerves-project/nerves_examples/tree/master/hello_leds/config) を参照してください。
```
Comment (riseshia) [inline ja/lessons/advanced/nerves.md]:
```suggestion
他のデバイスについては、[nerves_examples プロジェクトで対応する部分](https://github.com/nerves-project/nerves_examples/tree/master/hello_leds/config) を参照してください。
```

It's just english. same reason with https://github.com/elixirschool/elixirschool/pull/1889#discussion_r294030195

**Pull request merged**